src/test/resources/net/sf/jabref/logic/importer/fileformat/RisImporterTest3.bib

```diff
@@ -1,14 +1,14 @@
 @incollection{,
   author = {Barata, Catarina and Celebi, MEmre and Marques, JorgeS},
-  booktitle = {Dermoscopy Image Analysis},
   comment = {doi:10.1201/b19107-2},
   doi = {10.1201/b19107-2},
```

Comment:
I have seen that RisImporterTest3.ris specifies the both doi fields (M3 and DO) with the same doi string. Please move one of them to a different or new test file. This way the tests would have already showed that the DO field didn't import the doi.